### PR TITLE
Work and about section

### PR DESCRIPTION
**Pull request by mahdimahdawi**
The two last part of the portfolio project which was work section and about section are completed successfully.
Each section content image tags, heading tags, paragraph tags ....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,19 @@
...
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
```
Comment (cynthiainga) [inline README.md:32]: - [ ] Please kindly add this part for your README file, and also for each title give a short description, for example, for `setup` you can explain how to clone your repository locally and how to access the directory :upside_down_face: :upside_down_face:

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @mahdimahdawi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Descriptive title of PR :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please your Pull Request should have a short summary describing the introduced changes of this project in more detail, kindly update your Pull Request.
- [ ] Please kindly check the comments under the review for more details.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
The <a> tag changed to <hx> also the card padding done to placed inside the boxes, and the README file also updated.

**Review by danielufeli — APPROVED** [review]
Hi @mahdimahdawi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**